doc/changelog.md

```diff
@@ -2,6 +2,53 @@
 
 <h3>New features</h3>
 
+* Catalyst programs can now be used inside of a larger JAX workflow which uses JIT compilation,
+  automatic differentiation, and other JAX transforms.
+  [#96](https://github.com/PennyLaneAI/catalyst/pull/96)
+
+  Note that generally Catalyst should be used to JIT the entire workflow, but sometimes users may
+  wish to delegate only the quantum part of their workflow to Catalyst and let JAX handle the rest
+  (for example due to missing a feature or compatibility issue in Catalyst).

```

Comment:
Would you like to add this new support as a note to quick-start guideline?

Reply:
https://github.com/PennyLaneAI/catalyst/pull/96/commits/e191f8332b318146bea666912fe98b50712c8c22